frontend/app-development/config/routes.tsx

```diff
@@ -22,62 +23,55 @@ interface IRouteProps {
 }
 
 interface IRoute {
```

Comment:
Since we already is cleaning up this file. Should we remove the "I" and rename the interface to "RouterRoute" to make it more clear and descriptive? 😊 
```suggestion
interface RouterRoute {
```